packages/platform-ui/cypress/pages/auth/actions/signup.action.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="cypress" />
+import { User } from 'pages/common/interfaces/Auth';
+import { authSignupAction } from '../../../../src/store/auth';
```

Comment:
```suggestion
import { authSignupAction } from 'store/auth';
```
nit: it should be possible ^

Reply:
it did not work, I tried. I've added `paths` to `tsconfig.json` file and changed import to `@src/store/auth`